src/components/AddBookSection.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import dotenv from 'dotenv';
+import Aux from '../hoc/Auxilliary';
+import Toolbar from '../views/Navigation/ToolBar/ToolBar';
+import AddBookSectionView from '../views/AddBookSectionView';
+import getUserData from '../utils/jwtHandler';
+
+
+dotenv.config();
+const addBookSection = (props) => {
+    const token = localStorage.getItem('token');
+    const { email, isAdmin } = getUserData(token);
+    return !isAdmin? 
+     (<h1>Here we go with the dashboard for borrowers!</h1>)
```

Comment:
@blaise82 Thank you for taking the time to comment. Here I am trying to decode the token the user has used to login and if he/she is admin, the system redirect him to the admin dashboard otherwise to normal users dashboard(In our case we don't yet have this page!).

Let me know if you have any suggestions on how we can tweak this?